### PR TITLE
COMP2-02 - Governance-related Events

### PR DESCRIPTION
**Pull request by Hyodar**
# COMP2-02 - Governance-related Events

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] New Bot
- [ ] Fix a bot error
- [ ] Update a bot logic/test/documentation
- [ ] Code style update (formatting, renaming)
- [ ] Other (please describe):

## New Bot PR readiness

- [x] `npm install` command works (i.e. package-lock is updated properly)
- [x] The `name` and `description` fields of `package.json` describe the bot properly
- [x] All the libraries listed in `package.json` are being used
- [x] All the development libraries are in `devDependencies` section of `package.json`
- [x] The bot README is updated
- [x] The bot README sufficiently describes what the bot does
- [x] All the alerts that the bot emits are listed in the README
- [x] The txn/block examples listed in the README generate the expected findings
- [x] Code is easy to follow and/or properly commented
- [x] Bot contains tests
- [x] All tests pass when running `npm test`
- [x] Tests are not making real network calls (i.e. network interaction is properly mocked)
- [x] Tests cover all the paths (i.e. negative & positive test cases)
- [x] When the tests complete, no warning/errors are shown
- [x] `npm start` works without errors when executed using a jsonRpcUrl for each of the networks specified in the README
- [x] The networks specified in the README and the networks are listed in the `package.json` are the same
- [x] The code does what the README describes
- [ ] Network calls are properly being cached (if duplicated calls can occur)
- [x] A performance review has been done (e.g. `Promise.all` is used whenever possible)
- [x] The findings fields are filled out correctly
- [x] Findings `protocol` is specified (if the bot is for a specific one)
- [x] `metadata` of the findings contains all the information needed.
- [x] The SDK methods are being used appropriately (e.g. `filterLog`, `filterFunction`, `getEthersProvider`)
- [x] `ABI`s match the contracts used / `args` used from `LogDescriptions` exists in the `ABI`s
- [x] The bot follows Forta recommended [best practices](https://docs.forta.network/en/latest/best-practices/)
- [x] Bot can recover from failed error calls when they can fail (e.g. errors are being caught on calls that can fail)
- [x] `repository` is added in the `package.json`

### CODE REVIEW

File: CompoundV3-Bots/Governance-Events/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  - Severity is always set to "Medium"
+  - Type is always set to "Info"
+  - Metadata:
+    - `chain`: Network chain ID or name
+    - `comet`: Address of the related Comet contract
+    - `supplyPaused`: Status of supply pausing
+    - `transferPaused`: Status of supply pausing
+    - `withdrawPaused`: Status of supply pausing
+    - `absorbPaused`: Status of supply pausing
+    - `buyPaused`: Status of supply pausing
```
Comment (yasmine9) [inline CompoundV3-Bots/Governance-Events/README.md:28]: All descriptions are set to `supply pausing`, please update them to match the action

File: CompoundV3-Bots/Governance-Events/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "comp2-2",
+  "version": "0.0.1",
+  "description": "Forta bot that detects some events related to governance actions on Compound v3 Comet contracts",
+  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/CompoundV3-Bots/Protocol-Reserves",
```
Comment (yasmine9) [inline CompoundV3-Bots/Governance-Events/package.json:5]: Please update the repository field to the correct folder

File: CompoundV3-Bots/Governance-Events/src/agent.spec.ts [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+
+const network = Network.MAINNET;
+
+const DEFAULT_CONFIG: AgentConfig = {
+  [network]: {
+    cometAddresses: COMET_ADDRESSES,
+  },
+};
+
+describe("Bot Test Suite", () => {
```
Comment (yasmine9) [inline CompoundV3-Bots/Governance-Events/src/agent.spec.ts:106]: Just a small detail, I think it's better to include the bot name here for a more specific description, wdyt?

**Review by yasmine9 — CHANGES_REQUESTED** [review]
LGTM! minor changes to be addressed

File: CompoundV3-Bots/Governance-Events/README.md (hunk 1 above)
Comment (Hyodar) [reply CompoundV3-Bots/Governance-Events/README.md]: Good catch, fixed on [165698a](https://github.com/NethermindEth/Forta-Agents/pull/371/commits/165698a8b3e69086bf503179dc871bb2fd347501).

File: CompoundV3-Bots/Governance-Events/package.json (hunk 2 above)
Comment (Hyodar) [reply CompoundV3-Bots/Governance-Events/package.json]: Another good catch, fixed on [aa48e56](https://github.com/NethermindEth/Forta-Agents/pull/371/commits/aa48e5671851f266105a45fea1c639930dbbe716).

File: CompoundV3-Bots/Governance-Events/src/agent.spec.ts (hunk 3 above)
Comment (Hyodar) [reply CompoundV3-Bots/Governance-Events/src/agent.spec.ts]: Sure, agreed, done on [6b2edcb](https://github.com/NethermindEth/Forta-Agents/pull/371/commits/6b2edcb19f2a0dc726cbba59dd9b606c875c8b37).

**Review by yasmine9 — APPROVED** [review]

**Pull request merged**